### DISCUSSION TITLE
Why does python.exe use my microphone if this extenstion is installed?

### DISCUSSION THREAD

**Discussion by HacksoreDE** [Bing-su/adetailer#288] [Q&A] [answered]
I deleted all my extensions and reinstalled them one by one to see which one is using it. Seems like it's this one. Now i want to know why it is using it and for what purpose.

**Comment by Bing-su** [comment, marked as answer]
mediapipe와 관련된 현상으로 알고있습니다. adetailer는 오디오와 관련된 기능은 사용하지 않습니다.

**Reply by HacksoreDE** [reply]
Alright, thank you for the answer. Can this be fixed or will this always be a problem?

**Reply by Bing-su** [reply]
Well, I guess it's up to the [Google](https://developers.google.com/mediapipe) to do.
